src/components/command-bar/commands/search/components/no-results-message/index.tsx

```diff
@@ -3,14 +3,56 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+import { ReactElement } from 'react'
+import { SearchableContentType } from 'contexts'
 import Text from 'components/text'
+import { useCommandBar } from 'components/command-bar'
 import s from './no-results-message.module.css'
 
-const NoResultsMessage = () => {
+interface NoResultsMessageProps {
+	tabsWithResults: {
+		type: SearchableContentType
+		heading: string
+		icon: ReactElement<React.JSX.IntrinsicElements['svg']>
+	}[]
+	currentTabHeading: string
+}
+
+/**
+ * Renders a "no results" message, that optionally directs visitors
+ * to other content-type tabs with results, if such results are present.
+ */
+function NoResultsMessage({
```

Comment:
I've updated the `NoResultsMessage` component to render information on which other tabs have results. If no other tabs have results, we have a simpler "no matches" message. 

I've included content type icons here to hopefully add a bit of visual callback to the tabs themselves - cc @heitzke, let me know what you think 🙇‍♂️ Easy to remove if we'd prefer to omit them.